lib/cs_guide/resources/venue.ex

```diff
@@ -9,8 +9,14 @@ defmodule CsGuide.Resources.Venue do
 
   schema "venues" do
     field(:venue_name, :string)
+    field(:description, :string)
+    field(:address1, :string)
+    field(:address2, :string)
     field(:postcode, :string)
+    field(:website, :string)
     field(:phone_number, :string)
+    field(:facebook, :string)
```

Comment:
what is the _validation_ or _sanitisation_ being applied to the `:facebook` or `:instagram` fields? are they just "free text"? how do we protect against XSS?